sushi-config.yaml

```diff
@@ -61,7 +61,7 @@ pages:
   artefacts.md:
     title: FHIR artefacts      
     profiles-and-extensions.md:
-    # capability-statements.md:
+    capability-statements.md:
     search-parameters-and-operations.md:
   terminology.md:
```

Comment:
update the server actor to reference the capabilitystatement in the capabilities element  - this comment is not about this file, it is raised here to enable inline tracking.

Reply:
Fixed.